### PR TITLE
Feature28 fix disable button

### PR DESCRIPTION
**Pull request by xManuel007**
# Description

What did you implemented/Why did you implemented this:

- We fix the bug of the employee´s view, the button now can disable a enable an employee
- Also we added some style to the button
- And we update the readme
- now the employee will sort by if is an active employee or its disable

## Screenshots

image

### CODE REVIEW

**Review by AangelF — APPROVED** [review]

**Review by DiegoCoronaBarragan — APPROVED** [review]

**Review by oscaregr — APPROVED** [review]

File: app/assets/images/recycling.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="48" width="48"><path d="m18.4 18.4 4.45-7.35-2.95-4.9q-.6-1-1.725-1t-1.725 1l-4.9 8.15ZM37.75 32l-4.45-7.4 6.95-4 3.2 5.35q.55.85.6 1.9.05 1.05-.45 1.95-.5 1-1.475 1.6-.975.6-2.125.6ZM32 46l-8-8 8-8v4h9.5l-2.9 5.8q-.55 1-1.5 1.6-.95.6-2.1.6h-3Zm-19.35-4q-1 0-1.825-.525T9.6 40.1q-.4-.8-.375-1.675.025-.875.475-1.625l1.7-2.8H20v8ZM7.7 36.3l-3.25-6.5q-.45-.9-.425-1.925.025-1.025.575-1.925l.8-1.35L2 22.55l10.95-2.75 2.75 11-3.45-2.1Zm27-17.1-10.95-2.75 3.45-2.05L20.95 4H28q1.05 0 1.975.525T31.45 5.95l2.6 4.35 3.4-2.1Z"/></svg>
```
Comment (juperameza) [inline app/assets/images/recycling.svg:1]: New line at the end of this file

File: app/views/employees/index.html.erb [hunk 2]
```diff
@@ -35,29 +36,28 @@
...
+              turbo_method: :delete,
+              turbo_confirm: "Confirm Action"
+            } %>
         </div>
       </div>
     </div>
   <% end %>
-</div>
-
+</div>
```
Comment (juperameza) [inline app/views/employees/index.html.erb:63]: New line at the end of this file

**Review by juperameza — CHANGES_REQUESTED** [review]

File: app/views/employees/index.html.erb (hunk 2 above)
Comment (xManuel007) [reply app/views/employees/index.html.erb]: done

File: app/assets/images/recycling.svg (hunk 1 above)
Comment (xManuel007) [reply app/assets/images/recycling.svg]: done

**Review by juperameza — APPROVED** [review]
LGTM 
Nice blank lines

**Pull request merged**